### PR TITLE
New version of qrcode exporter

### PR DESCRIPTION
**Pull request by gre**

### CODE REVIEW

File: src/components/SettingsPage/sections/Tools.js [hunk 1]
```diff
@@ -30,6 +31,7 @@ class TabProfile extends PureComponent<*, *> {
         <Box flow={2}>
           <QRCodeExporter />
         </Box>
+        <Box style={{ textAlign: 'right' }}>{liveCommonPkg.version}</Box>
```
Comment (meriadec) [inline src/components/SettingsPage/sections/Tools.js]:
```
<Box align="right">
```
:p

Comment (gre) [reply src/components/SettingsPage/sections/Tools.js]: it doesn't work

Comment (gre) [reply src/components/SettingsPage/sections/Tools.js]: rollbacking. btw not very important because new design to come later.

Comment (meriadec) [reply src/components/SettingsPage/sections/Tools.js]: my bad, it's `textAlign="right"`, the `align` prop is bound to `align-items`

Comment (meriadec) [reply src/components/SettingsPage/sections/Tools.js]: nvm

**Pull request merged**